odoo/addons/test_http/controllers.py

```diff
@@ -148,3 +149,19 @@ def touch(self):
     @http.route('/test_http/json_value_error', type='json', auth='none')
     def json_value_error(self):
         raise ValueError('Unknown destination')
+
+    @http.route('/test_http/hide_errors/decorator', type='http', auth='none')
+    @reraise_x_as(AccessError, as_=werkzeug.exceptions.NotFound())
+    def hide_errors_deco(self, error):
```

Comment:
These method names are written a grand total of once each, I don't think the abbrevitating is warranted.